pkg/registry/interface.go

```diff
@@ -8,7 +8,10 @@ import (
 
 type Load interface {
 	AddOperatorBundle(bundle *Bundle) error
+	AddBundlePackageChannels(manifest PackageManifest, bundle Bundle) error
 	AddPackageChannels(manifest PackageManifest) error
+	RmPackageName(packageName string) error
+	ClearNonDefaultBundles(packageName string) error
```

Comment:
non-blocking: I think these should probably be separate interfaces. we can refactor later.